src/app/myworkflows/myworkflows.service.ts

```diff
@@ -98,7 +98,7 @@ export class MyWorkflowsService extends MyEntriesService<Workflow, OrgWorkflowOb
    * @param {number} id  The user ID
    * @memberof MyBioWorkflowsService
    */
-  getMyBioWorkflows(id: number): void {
+  getMyBioWorkflows(id: number, showEntry: boolean): void {
```

Comment:
Does this function (and `getMyEntries`) need to change? From what I can tell, it's not used in your other files

Reply:
Oops this was a change from a while back from when I was still using this function to get the list, good catch!